.gitignore

```diff
@@ -1 +1,5 @@
+# SPDX-License-Identifier: CC0-1.0
+# SPDX-FileCopyrightText: Contributors to the gnome-nvidia-extension project.
```

Comment:
Just a nitpick, but since you're effectively releasing the file into the public domain, there is no copyright on it (and line 2 can be removed).